pkg/cmd/server/origin/aggregator.go

```diff
@@ -72,14 +75,18 @@ func (c *MasterConfig) createAggregatorConfig(genericConfig genericapiserver.Rec
 	}
 
 	aggregatorConfig := &aggregatorapiserver.Config{
-		GenericConfig: &genericConfig,
+		GenericConfig: &genericapiserver.RecommendedConfig{
+			Config:                genericConfig,
+			SharedInformerFactory: externalInformers,
```

Comment:
why not instatiate outside of createAggregatorConfig ?

Reply:
only a nit

Reply:
@sttts this is what kube does, wanted to copy their pattern.